config/config.go

```diff
@@ -701,11 +702,17 @@ func normalizeEndpoint(u string) string {
 	// lowercase.
 	u = strings.ToLower(u)
 
+	isHttp := strings.HasPrefix(u, "http://")
+
 	// if URL has no scheme, prefix it with the default scheme.
-	if !strings.HasPrefix(u, "https://") && !strings.HasPrefix(u, "http://") {
+	if !strings.HasPrefix(u, "https://") && !isHttp {
 		u = "https://" + u
 	}
 
+	if isHttp {
+		fmt.Println(color.YellowString("[Warning] Endpoint URL uses an insecure protocol"))
```

Comment:
```suggestion
		fmt.Println(color.YellowString("Warning: endpoint URL uses an insecure protocol"))
```

Little style alignment.